internal/cli/import/config.go

```diff
@@ -1,8 +1,29 @@
 package cli
 
-// Put here the logic for parsing the JSON config file
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+)
 
 type Config struct {
 	Auth0Domain                 string   `json:"AUTH0_DOMAIN"`
 	Auth0KeywordReplaceMappings struct{} `json:"AUTH0_KEYWORD_REPLACE_MAPPINGS"`
 }
+
+func getConfig(path string) *Config {
```

Comment:
This should return the error as well, instead of printing it: 
<img width="330" alt="Screen Shot 2022-03-09 at 18 54 48" src="https://user-images.githubusercontent.com/5055789/157542919-6eecfc32-10bd-442e-8ed5-9c4b7ee104d2.png">
The error always goes last.